source/installation-guide/installing-wazuh-manager/linux/amazon/wazuh_server_sources_amazon.rst

```diff
@@ -92,24 +92,13 @@ Installing Wazuh manager
 
       # service wazuh-manager status
 
-Installing Wazuh API
---------------------
-
-1. NodeJS >= 4.6.1 is required in order to run the Wazuh API. If you do not have NodeJS installed or your version is older than 4.6.1, we recommend you add the official repository as this has more recent versions.
-
-    .. code-block:: console
-
-      # curl --silent --location https://rpm.nodesource.com/setup_10.x | bash -
-      # yum -y install nodejs
-      # npm config set user 0
-
-2. Download and execute the installation script:
+.. note:: You can also run an :ref:`unattended installation <unattended-installation>` for the Wazuh manager and API.
 
-  .. code-block:: console
+.. versionadded:: 4.0.0
 
-      # curl -s -o install_api.sh https://raw.githubusercontent.com/wazuh/wazuh-api/v|WAZUH_LATEST|/install_api.sh && bash ./install_api.sh download
+The Wazuh API will be installed along the Wazuh manager by default. No extra steps or requirements are needed to install it.
 
-3. Once the process is complete, you can check the service status with:
+To check the API service status use the following command:
```

Comment:
Please, apply the Wazuh manager status review in all documents.

Reply:
To be changed in corresponding issue wazuh/wazuh#5824